### PR TITLE
add proposals for WLCG

### PR DESCRIPTION
**Pull request by maany**
Hi admins,

Here are two proposals from WLCG that we wish to incorporate with HEP-SF during GSoC 2019.

### CODE REVIEW

File: _gsocproposals/2019/proposal_WLCGansible.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+**Requirements**
+
+* Familiarity and prior experience with Ansible.
+* Programming experience in Python is a plus.
+* Linux string fundamentals.
+* Git/Agile development methodology.
+
+
+
+**Mentors**
```
Comment (agheata) [inline _gsocproposals/2019/proposal_WLCGansible.md]: Please add the mentors with their affiliations to gsoc/2019/mentors.md

File: _gsocproposals/2019/proposal_WLCGansible.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: Python Components for the SIMPLE Grid Framework
```
Comment (agheata) [inline _gsocproposals/2019/proposal_WLCGansible.md]: The 2 projects have identical titles, can you please change to be more specific to each proposal

File: _gsocproposals/2019/proposal_WLCGansible.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+layout: gsoc_proposal
+project: WLCG
+year: 2019
+organization:
+  - CERN
+---
+
+## Description
```
Comment (agheata) [inline _gsocproposals/2019/proposal_WLCGansible.md]: The 2 projects have identical description. The general part can be indeed common, but the second part is maybe more useful to describe the specific task for one proposal or another.

Comment (maany) [reply _gsocproposals/2019/proposal_WLCGansible.md]: Indeed. I have updated the title and description of the Ansible project. Thanks for pointing this out.

File: _gsocproposals/2019/proposal_WLCGansible.md (hunk 1 above)
Comment (maany) [reply _gsocproposals/2019/proposal_WLCGansible.md]: done

**Comment by maany** [comment]
@agheata, could you check the new commit that addresses your review?

File: gsoc/2019/mentors.md [hunk 4]
```diff
@@ -105,3 +105,5 @@ layout: plain
 * Frank Wuerthwein [fkw@ucsd.edu](mailto:fkw@ucsd.edu) UCSD
 * Stefan Wunsch [stefan.wunsch@cern.ch](mailto:stefan.wunsch@cern.ch) CERN, KIT
 * Omar Zapata [Omar.Zapata@cern.ch](mailto:Omar.Zapata@cern.ch) OProject
+* Mayank Sharma [mayank.sharma@cern.ch](mailto:mayank.sharma@cern.ch) CERN
```
Comment (JavierCVilla) [inline gsoc/2019/mentors.md]: Can you please add them keeping the alphabetical order?

**Comment by agheata** [comment]
I'll do the re-ordering in a separate PR. Merging.

**Pull request merged**